lib/Doctrine/DBAL/Schema/Visitor/ViewVisitor.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Doctrine\DBAL\Schema\Visitor;
+
+use Doctrine\DBAL\Schema\View;
+
+/**
+ * Visitor that can visit views.
+ *
+ * @link   www.doctrine-project.org
```

Comment:
Please remove.(see https://github.com/doctrine/dbal/pull/3318).

Reply:
Does it make sense to leave `Visitor that can visit views` as it is?